website/docs/registry/providers/docs.mdx

```diff
@@ -181,6 +184,8 @@ After publishing this provider version, its page on the Terraform Registry would
   * example_instance
 * Functions
   * parse_instance_id
+* Ephemeral Resources
+  * example_auth_token
```

Comment:
If I want to be extra-🤓 the correct order would be Functions, Ephemeral Resources and then Resources followed by Data Sources (it already wasn't correct before your change as the Functions header was moved to the top in a follow-up PR after the initial implementation which probably was forgotten to reflect here).
However, I don't have a strong opinion on whether this is important to reflect at all 😄 But I just wanted to call it out 🤓

Reply:
(this would apply to the other examples below as well)